services/storage-users/pkg/config/config.go

```diff
@@ -171,7 +171,7 @@ type Events struct {
 type Cache struct {
 	Store    string   `yaml:"store" env:"OCIS_CACHE_STORE_TYPE;STORAGE_USERS_CACHE_STORE_TYPE;STORAGE_USERS_CACHE_STORE" desc:"Store implementation for the cache. Supported values are 'memory' (default), 'redis', 'redis-sentinel', 'nats-js', and 'etcd'. See the text description for details."`
 	Nodes    []string `yaml:"nodes" env:"OCIS_CACHE_STORE_ADDRESS;STORAGE_USERS_CACHE_STORE_ADDRESS;STORAGE_USERS_CACHE_NODES" desc:"A comma separated list of addresses to access the configured store. This has no effect when the 'memory' store is configured. Note that the behaviour how addresses are used is dependent on the library of the configured store."`
-	Database string   `yaml:"database" env:"STORAGE_USERS_CACHE_DATABASE" desc:"(optional) The database name the configured store should use. This has no effect when a 'memory', 'redis' or 'redis-sentinel' store is configured."`
+	Database string   `yaml:"database" env:"STORAGE_USERS_CACHE_DATABASE" desc:"The database name the configured store should use. This has no effect when a 'memory', 'redis' or 'redis-sentinel' store is configured."`
```

Comment:
Isn't it `optional` any more? Or is it only `not optional` for certain types?

Reply:
It has an effect on all stores. It can be left blank, but then data will be written to the same database, which, which using the same Redis instance (or in memory cache) might lead to key collisions. The database is used as a prefix for the key.

The default config should not be empty and admins might want to customize which database is used ... but it should not really be necessary to touch this.